wordpress-mobile/WordPress-iOS
# Split RN patches to separated files

Fixes #https://github.com/wordpress-mobile/gutenberg-mobile/issues/2410

To test:

* Check the code:

1. Run `pod install`
2. Go to `Pods/React-Core/React/Views/RCTShadowView.m` and check if there is code:
	`while (shadowView && shadowView != ancestor) `
3. Go to `Pods/React-RCTActionSheet/RCTActionSheetManager.m` and check if there is code:

	```
		+    if ([disabledButtonIndices containsObject:@(localIndex)]) {
		+        [alertController.actions[localIndex] setEnabled:false];
		+    }
	```

* Check the app:

1. Run the app
2. Add new post
3. Add `Group` block
4. Inside a group and `Paragraph`
5. Inside a group add new `Group`
6. Repeat steps 4 and 5 at least one time
7. Expect: the app cannot crash

* Check the app:

1. Run the app
2. Add new post
3. Add `Buttons` block
4. Press `…` button
5. Expect `Move block left` and `Move block right` are disabled


PR submission checklist:
